overlord/snapstate/snapmgr.go

```diff
@@ -1067,6 +1071,64 @@ func (m *SnapManager) localInstallCleanup() error {
 	return osutil.UnlinkManyAt(d, filenames)
 }
 
+func MockEnsuredMountsUpdated(m *SnapManager, ensured bool) (restore func()) {
+	osutil.MustBeTestBinary("ensured snap mounts can only be mocked from tests")
+	old := m.ensuredMountsUpdated
+	m.ensuredMountsUpdated = ensured
+	return func() {
+		m.ensuredMountsUpdated = old
+	}
```

Comment:
no, it is used by other package afair? also we can't use testutil from production code, so unless we are in export_test.go no testutil.Backup for us. Life is hard.

Reply:
Move it into snapstatetest package and use testutil.Backup instead?

Reply:
can't do that either because that package can't touch the unexported flag. The code is fine as is really. testutil.Backup is convenient but not worth making the code a pretzel over it.

Reply:
Yea wasn't so worried about the testutil.Backup, just wanted to avoid Test code in production code